async/js/async-await.js

```diff
@@ -0,0 +1,47 @@
+const peopleList = document.getElementById('people');
+
+async function getAstroInfo(url) {
+    try {
+        const response = await fetch(url);
+        return await response.json();
+    } catch (error) {
+        throw error;
+    }
```

Comment:
This is fine.
I love to see one line try/catch's if possible.
We can talk about it more.